package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "django-dynamic-attachments",
+  "version": "3.5.0",
+  "description": "Frontend for a Django application for handling file uploads and attaching them to arbitrary models.",
+  "files": [
+    "attachments/static/attachments/**"
+  ],
+  "main": "attachments/static/attachments/js/attachments.js",
+  "directories": {
+    "doc": "docs"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
Should this be merged?

Reply:
Ah nah it shouldn't. That is auto generated by "npm init". Removed it